src/Symfony/Component/Workflow/Dumper/PlantUmlDumper.php

```diff
@@ -208,7 +208,9 @@ private function getState(string $place, Definition $definition, Marking $markin
 
         $description = $workflowMetadata->getMetadata('description', $place);
         if (null !== $description) {
-            $output .= \PHP_EOL.$placeEscaped.' : '.str_replace("\n", ' ', $description);
+            foreach (array_filter(explode("\n", $description)) as $line) {
+                $output .= \PHP_EOL.$placeEscaped.' : '.$line;
```

Comment:
```suggestion
                $output .= "\n".$placeEscaped.' : '.$line;
```

PHP_EOL is platform dependent